src/main/java/useCases/MakeBotMove.java

```diff
@@ -2,21 +2,21 @@
 
 import entities.GameState;
 import entities.Player;
+import entities.BotPlayer;
+import java.util.List;
 
 public class MakeBotMove {
-
-    //Should only be called when toMove is not 0 (so toMove is a bot player).
+    // Should only be called when toMove is not 0 (so toMove is a bot player).
     public void makeBotPlay(GameState gameState) {
-        Player botPlayer = gameState.getPlayers().get(gameState.getToMove());
-
-        //Do logic here to get the card this bot wants to play
-        int toPlay = 1; //Delete this later
+        Player player = gameState.getPlayers().get(gameState.getToMove());
 
-        if (toPlay == -1) {
+        if (player instanceof BotPlayer) {
```

Comment:
Removing instanceof would be optimal. For instance, (<< ha ha) if Player has a makeMove() method that gets the presenter to update the screen, botPlayer could override it to make its move automatically